app/serializers/v1/fee_serializer.rb

```diff
@@ -16,8 +16,8 @@ def serialize
           lago_item_id: model.item_id,
           item_type: model.item_type,
         },
-        pay_in_advance: model.pay_in_advance,
-        invoiceable: model.charge&.invoiceable || false,
+        pay_in_advance: calculate_pay_in_advance,
+        invoiceable: model.charge? ? model.charge&.invoiceable : true,
```

Comment:
Maybe this one could be exported into a proper `invoiceable` method in this serializer`?